zc_install/includes/classes/class.systemChecker.php

```diff
@@ -264,6 +264,13 @@ public function dbVersionCheckConfigDescription($db, $dbPrefix, $parameters)
   }
   public function checkWriteableDir($parameters)
   {
+    if ($parameters['makeWritable'] === true) {
+      foreach(['666', '766'] as $perm) {
```

Comment:
Pinging @barco57 for thoughts on a safe progression of perm levels to attempt to set here.  Concern is granting too much, but also granting too little such that one locks oneself out. And of course there's the desire to be as broadly-compatible with various hosting server configurations as possible. 

Feedback welcome.